client/test/app/hearings/components/VirtualHearings/Timezone.test.js

```diff
@@ -130,9 +130,9 @@ describe('Timezone', () => {
   });
 
   test('Dropdown displays correct times based on props time and roTimezone', () => {
-    const time = HEARING_TIME_OPTIONS[0].value;
+    const time = "08:15 am";//HEARING_TIME_OPTIONS[0].value;
     const roTimezone = 'America/Los_Angeles';
-    const dateTime = moment.tz(time, 'HH:mm', roTimezone)
+    const dateTime = moment.tz(time, 'h:mm a', roTimezone)
     const roTzValueToLabelMapping = invert(TIMEZONES)
```

Comment:
I saw these pop up in my local linter:
```suggestion
    const dateTime = moment.tz(time, 'h:mm a', roTimezone);
    const roTzValueToLabelMapping = invert(TIMEZONES);
```